api/players/overview/types.ts

```diff
@@ -85,12 +85,26 @@ export interface ProfileOverview {
 		}
 		altPosition: {
 			name: string
-			gridPos: number
+			gridPos: Int32
+		}
+		secondAltPosition: {
+			name: string
+			gridPos: Int32
 		}
 		position: {
 			name: string
 			gridPos: number
 		}
+		languages: {
+			id: string
+			name: string
+			countryFlagCode: string
+		}[]
+		citizenships: {
+			id: string
+			name: string
+			countryFlagCode: string
+		}[]
```

Comment:
countries flags are created from their according codes not some flag specific code
```suggestion
		languages: {
			id: string
			name: string
			code: string
		}[]
		citizenships: {
			id: string
			name: string
			code: string
		}[]
```